files/image_config/caclmgrd/caclmgrd

```diff
@@ -396,6 +396,7 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                 acl_rules = {}
 
                 for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.iteritems():
+                    rule_props = {k.upper(): v for k,v in rule_props.items()}
```

Comment:
Since this script is still using Python 2 in 201911, I think you should change this to `rule_props.iteritems()`, as `.items()` in Python 2 can consume more memory if the dict is large.

Reply:
addressed in next commit.